### DISCUSSION TITLE
Problem to open dakota

### DISCUSSION THREAD

**Discussion by leobiscaino** [snl-dakota/dakota#104] [Q&A]
Hi, I'm using dakota GUI and I have any problems. First, I set the environments variables, like the picture shows, and them I installed all required libraries. How you can see, dakota --v is not working, and when I use the gui, this problem shows up. 
![Captura de tela de 2024-03-05 14-39-50](https://github.com/snl-dakota/dakota/assets/93631643/1a7e1c72-7008-45e7-8ecf-96ba579420d8)
![Captura de tela de 2024-03-05 14-38-44](https://github.com/snl-dakota/dakota/assets/93631643/7e796f07-1816-4f74-ba15-e5994e4e7ea4)
![Captura de tela de 2024-03-05 14-37-57](https://github.com/snl-dakota/dakota/assets/93631643/cd541d8c-5406-4a40-aec9-0e9c1c1b5b0b)
![Captura de tela de 2024-03-05 14-37-19](https://github.com/snl-dakota/dakota/assets/93631643/4f8bf058-29ef-48a1-b7ee-a66bb305b89f)

**Comment by briadam** [comment]
In your shell config you reference /home/leobiscaino/Downloads/dakota. In Downloads, is dakota a symlink to dakota-6.19.0-public-rhel7.Linux.x86_64-guid_cli? If not, you'd need to set your path to that directory. Once setting those PATHs, you'll need to source your config or restart your shell.

**Reply by leobiscaino** [reply]
Yes,It is the dakota that you typed, I just change the name like the instruction sad.  I followed this link: https://dakota.sandia.gov/installation/

**Reply by leobiscaino** [reply]
I just dont change de C shell, because I don't have knowledge
